presto-main/src/test/java/com/facebook/presto/sql/planner/TestDynamicFilter.java

```diff
@@ -439,6 +439,7 @@ public void testNonPushedDownJoinFilterRemoval()
                 "SELECT 1 FROM part t0, part t1, part t2 " +
                         "WHERE t0.partkey = t1.partkey AND t0.partkey = t2.partkey " +
                         "AND t0.size + t1.size = t2.size",
+                noJoinReordering(),
```

Comment:
is this unrelated?

Reply:
Yes. I noticed that the [cherry-picked](https://github.com/trinodb/trino/pull/2659/files#diff-30da09fcaa15292108aeb231906067c6e7bc99cff571317ab506881e93456c95) code had this added to ensure that we always run this test with no join ordering. [This was missed](https://github.com/prestodb/presto/pull/15057/files#diff-5cbb8ede5399d40423b5c0b3a83df240c523d6841d055cf97224c4e4a12eb4a3) when we pulled it into Presto.